test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -22,3 +22,52 @@
         (Get-Module -Name $moduleName).Name | Should Be $moduleName
     }
 }
+
+Describe "Import-Module with ScriptsToProcess" -Tags "CI" {
+    
+    BeforeAll {
+        $moduleRootPath = Join-Path $TestDrive 'TestModules'
+        New-Item $moduleRootPath -ItemType Directory -Force | Out-Null
+        Push-Location $moduleRootPath
```

Comment:
I chose push location specifically because I didn't want to deal with full paths. :)